pagerdutyplugin/data_source_pagerduty_standards_resources_scores.go

```diff
@@ -0,0 +1,118 @@
+package pagerduty
+
+import (
+	"context"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type dataSourceStandardsResourcesScores struct {
+	client *pagerduty.Client
+}
+
+var _ datasource.DataSource = (*dataSourceStandardsResourcesScores)(nil)
+
+func (d *dataSourceStandardsResourcesScores) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = "pagerduty_standards_resources_scores"
+}
+
+func (d *dataSourceStandardsResourcesScores) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"ids": schema.ListAttribute{
+				ElementType: types.StringType,
+				Required:    true,
+			},
+			"resource_type": schema.StringAttribute{Required: true},
```

Comment:
Please add input validation for available options. Currently only `technical_services`.